training/README.md

```diff
@@ -0,0 +1,125 @@
+# Training
+
+This section explains how to reproduce the T0 training: a massively multitask fine-tuning using TPUs and Mesh Tensorflow. These are the steps we used in the paper for training and evaluation.
+
+We also release code to replicate this training in PyTorch (see [Replicating the training in PyTorch](#replicating-the-training-in0-PyTorch)).
+
+Before starting, please make sure you have installed the dependencies listed in [`requirements.txt`](https://github.com/bigscience-workshop/t-zero/blob/master/requirements.txt).
+
+## Data pre-processing
+
+The first step is to pre-process and cache the data. We used [SeqIO](https://github.com/google/seqio), a library for processing sequential data to be fed into downstream sequence models. Essentially, SeqIO tokenizes (and caches) the input/target pairs and handles the training and evaluation mixtures.
+
+You can pre-process a given task with the following command:
```

Comment:
I would say "cache" not "pre-process" here, "preprocess" has a specific and different meaning in seqio